todo_app/browser_tests/chrome_test.py

```diff
@@ -41,12 +46,16 @@ def test_app():
     
     # Tear Down
     thread.join(1)
-    delete_trello_board(test_board_id)
+    delete_trello_board(test_board_id, api_key, api_token)
 
 
 @pytest.fixture(scope='module') 
 def driver():
-    with webdriver.Chrome() as driver:
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    opts.add_argument('--disable-dev-shm-usage')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
chromedriver.exe should be added to .gitignore.

Also this can just be `webdriver.Chrome(options=opts)` which will check both the PATH and the current directory for the chromedriver.exe

Reply:
I tried it this way, but it won't run in Docker.
Therefore I have left as is for now, but if you can advise why Docker is not finding the driver in the PATH, would be grateful!